### ISSUE TITLE
Split chat buttons for easier fast-forwarding of rolls

### ISSUE THREAD

**Issue by theripper93** [foundryvtt/dnd5e#2994]
Split chat buttons so that by default they will fast forward but by pressing the section on the left the dialog will show.

Most times you don't really need the roll dialog and having to hold shift to skip it is an extra step in most rolls.

![image](https://github.com/foundryvtt/dnd5e/assets/1346839/ef074b99-e19a-40e4-bea3-fa509f090323)

**Comment by schultzcole** [comment]
To me this seems like a reasonable compromise on the "show roll dialog by default" vs "fast forward by default" debate.

**Comment by arbron** [comment]
I'd probably vote that the larger default button bring up the config window and the smaller one be for fast forward:

![303701377-ef074b99-e19a-40e4-bea3-fa509f090323](https://github.com/foundryvtt/dnd5e/assets/19979839/3865b381-c15e-4c7b-bc5e-7d026639d3ee)

One additional thing we could do it detect what keys the user is pressing, and update the button appropriately. So if any of the three keys are held down, the fast forward control expands to be the full width of the button. If alt/option is held down we show the advantage dice symbol, and if control/command is held down we show the disadvantage symbol.

**Comment by theripper93** [comment]
I like the idea but i still think the Larger button should be the most common use case. In my opinion that is fast forwarding. Is there any prevous consideration as to why the dialog would be considered the most desired result most of the time?

**Comment by krbz999** [comment]
> Is there any prevous consideration as to why the dialog would be considered the most desired result most of the time?

People with modules that prevent non-fastforwarding all too often do not even know the roll config exists.

Those who do want FF being the default can still do so by whichever means (or just hold shift), while leaving the default as is means not *everyone* has to adapt.

**Comment by theripper93** [comment]
This is not really an answer to the question. If you have a split button with a dialog icon surely you'd know that the roll config exists. I'm asking why the default would be having the dialog, as most times, you don't actually need it

**Comment by MaxPat931** [comment]
because without a module providing Adv/Dis flags, there wouldnt be an option to select the appropriate option

**Comment by theripper93** [comment]
> because without a module providing Adv/Dis flags, there wouldnt be an option to select the appropriate option

That's not true, you simply click the dialog button as in the screenshot to bring up the roll with the configuration

**Comment by MotoMoto1234** [comment]
Can we also move all the dnd5e popups to the same general location as the foundry popups so that we aren't playing whack a mole on a 4k resolution monitor wearing out the mouse pad?

**Comment by krbz999** [comment]
That should probably be opened as a different issue. :)
